src/test/java/chess/piece/KnightTest.java

```diff
@@ -0,0 +1,49 @@
+package chess.piece;
+
+import chess.domain.board.File;
+import chess.domain.board.Rank;
+import chess.domain.board.Square;
+import chess.domain.piece.Camp;
+import chess.domain.piece.Piece;
+import chess.domain.piece.PieceType;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+class KnightTest {
+    private static final Piece knight = PieceType.KNIGHT.createPiece(Camp.WHITE);
+    private static final Square source = new Square(File.E, Rank.FOUR);
+
+    @DisplayName("나이트 이동조건이 아닌 경우, 이동할 수 없다")
+    @Test
+    void cantMoveOverMovableDistance() {
+        Square target = new Square(File.E, Rank.ONE);
+        Assertions.assertThat(knight.canMove(source, target))
+                .isFalse();
+    }
+
+    @ParameterizedTest(name = "나이트는 {0}로 이동할 수 있다")
+    @MethodSource("squareProvider")
+    void canMoveTestWithMovableRankDistance(String name, Square target) {
+        Assertions.assertThat(knight.canMove(source, target))
+                .isTrue();
+    }
+
+    static Stream<Arguments> squareProvider() {
+        return Stream.of(
+                Arguments.arguments("e4에서 g3", new Square(File.G, Rank.THREE)),
```

Comment:
```suggestion
                Arguments.arguments("오른쪽 대각선 + 전진 ", new Square(File.G, Rank.THREE)),
```
이런 방식으로 어떻게 움직이는지 좀 더 구체화 해주면 좋겠네요!ㅎㅎㅎ